gw_eccentricity/eccDefinition.py

```diff
@@ -305,6 +313,13 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
         # called, these get set in that function.
         self.t_for_omega22_average = None
         self.omega22_average = None
+        # Approximate number of orbits derived using the phase of 22 mode
+        # assuming that a phase change 4pi occurs over an orbit.
+        self.approximate_num_orbits = ((self.phase22[-1] - self.phase22[0])
+                                       / (4 * np.pi))
+        # The following is updated to True when the waveform has enough number
+        # of orbits but a method can not find sufficient number of extrema.
+        self.probably_quasicircular = False
```

Comment:
Does this need to be in self? Why not just in the function where this is determined?

Reply:
Done